pyproprop/options.py

```diff
@@ -0,0 +1,98 @@
+"""A utility designed to work with processed properties with options.
+
+Processed properties allow the specification of a group of options that a user
+can then choose from. If an option from this group is chosen then an error is
+raised. Processed property options also allow for a default option to be
+specified as well as for options to be specified as unsupported. Additionally
+dispatchers can be built using these options so that a specific function or
+class handle can be linked to the option identifiers. This module implements a
+framework to provide all of these things in a clean and easy-to-use way that is
+designed with use alongside processed properties in mind.
+
+"""
+
+
+__all__ = ["Options"]
+
+
+from collections.abc import Sequence
+
+from .utils import (format_as_iterable, format_for_output)
+
+
+class Options:
```

Comment:
Class level docstring ideally should be included

Reply:
Addressed in 95139c72cd02c55b93494b9adfffe5890fd21267.